oss/ossSocket.hpp

```diff
@@ -25,136 +25,136 @@
 #define OSS_MAX_HOSTNAME NI_MAXHOST 
 #define OSS_MAX_SERVICENAME NI_MAXSERV 
 
-class _ossSocket {
```

Comment:
to better maintain and isolate modules we add namespace emeralddb/${module_name}.
all modules should be under the namespace of emeralddb